package.json

```diff
@@ -127,6 +127,7 @@
     "jquery": "^3.6.1",
     "jwt-decode": "^3.1.2",
     "md5": "^2.3.0",
+    "moment": "^2.29.4",
```

Comment:
Can this live under `devDependencies`? Do we use it in production code?

Reply:
Yes not needed. I have moved into ```devDependencies```